transaction/src/builder/manifest_builder.rs

```diff
@@ -715,6 +715,56 @@ impl ManifestBuilder {
         .0
     }
 
+    pub fn lock_fee_and_withdraw(
+        &mut self,
+        amount: Decimal,
+        resource_address: ResourceAddress,
+        account: ComponentAddress,
```

Comment:
Your change is consistent with the rest, but it feels a little confusing to me that the account is last because it doesn't match how you'd write the manifest line...

What are your thoughts on refactoring these methods on the manifest builder to put the account first? (I imagine CLion might be able to make that easy?)

Reply:
Looks like some existing methods are already taking `account` parameter as the last argument.

But, I like the idea of making them consistent with manifest, so up to change them all?